gradle/tests.gradle

```diff
@@ -141,7 +141,9 @@ tasks.test.configure {
 
 tasks.register("fastReplayTests", Test) {
   dependsOn(buildOldReplayTestsZkevmBin)
-
+    // Record zkevm.bin for CorsetValidator
+    environment.put("ZKEVM_BIN", System.getenv("ZKEVM_BIN") ?: "zkevm_for_old_replay_tests.bin")
```

Comment:
@letypequividelespoubelles Storing this in the environment here means that `GoCorsetValidator` will pick it up and direct `go-corset` to use this `bin` file for checking.